src/Core/Models/Table/Cipher.cs

```diff
@@ -41,7 +41,12 @@ public void SetNewId()
 
             try
             {
-                _attachmentData = JsonConvert.DeserializeObject<Dictionary<string, CipherAttachment.MetaData>>(Attachments);
+                _attachmentData = JsonConvert.DeserializeObject<Dictionary<string, CipherAttachment.MetaData>>(Attachments,
+                    new JsonSerializerSettings { DefaultValueHandling = DefaultValueHandling.Populate });
```

Comment:
This should no longer be necessary either w/ the recommended changes in `Data/CipherAttachment.cs`